auto_nag/scripts/configs/dom_navigation_round_robin.json

```diff
@@ -1,7 +1,6 @@
 {
   "fallback": "Andrew Overholt",
   "components": {
-    "Core::DOM: Navigation": "default",
     "Core::DOM: Window and Location": "default"
```

Comment:
This one should probably move to `DOM LWS` given https://bugzilla.mozilla.org/rest/config/component_teams/DOM%20LWS

Reply:
Let's keep it here for now though

Reply:
https://github.com/mozilla/relman-auto-nag/pull/1177